lib/chef/audit/runner.rb

```diff
@@ -79,11 +79,15 @@ def require_deps
         require 'rspec'
         require 'rspec/its'
         require 'specinfra'
+        require 'specinfra/helper'
+        require 'specinfra/helper/set'
         require 'serverspec/helper'
         require 'serverspec/matcher'
         require 'serverspec/subject'
         require 'chef/audit/audit_event_proxy'
         require 'chef/audit/rspec_formatter'
+
+        Specinfra::Backend::Cmd.send(:include, Specinfra::Helper::Set)
```

Comment:
It looks like this is a bug in Specinfra? https://github.com/serverspec/specinfra/blob/master/lib/specinfra/backend/cmd.rb calls `set` but doesn't include that set helper anywhere.

For some reason you cannot file issues against https://github.com/serverspec/specinfra, but it looks like it would be a simple PR.  Even if we include this change here I would like to see a PR opened against them.